lighthouse-core/gather/driver.js

```diff
@@ -477,11 +478,15 @@ class Driver {
     const waitForLoad = _options.waitForLoad || false;
     const disableJS = _options.disableJavaScript || false;
     const pauseAfterLoadMs = (_options.flags && _options.flags.pauseAfterLoad) || PAUSE_AFTER_LOAD;
+    let maxWaitMs = MAX_WAIT_FOR_FULLY_LOADED;
+    if (_options.flags && _options.flags.timeout) {
+      maxWaitMs = 1000 * _options.flags.timeout;
```

Comment:
ah i see. totally a style thing.. I'm just used to seeing the integers on the right side.

but good call on doing this for protection with +.

nbd either way